modules/table/src/main/java/org/apache/ignite/internal/table/distributed/command/PartitionCommand.java

```diff
@@ -18,23 +18,31 @@
 package org.apache.ignite.internal.table.distributed.command;
 
 import java.util.UUID;
-import org.apache.ignite.internal.schema.BinaryRow;
 import org.apache.ignite.raft.client.WriteCommand;
 import org.jetbrains.annotations.NotNull;
 
 /**
- * The command inserts a row.
+ * RAFT partition transactional command.
```

Comment:
Should we specify here that a command is a RAFT command?

Reply:
Fixed.